content/en/product_analytics/journeys/_index.md

```diff
@@ -31,11 +31,11 @@ With [funnel analysis][1], you can understand the end-to-end conversion of a sin
 
 {{< img src="/product_analytics/journeys/pa-conversions.png" alt="View conversion rate values over time in graphs ">}}
 
-You can view [conversion rates][3] over time by showing the conversion rate value in graphs.
+You can view [conversion rates][3] over time by showing the conversion rate value in graphs. Note that only completed RUM sessions contribute to conversion rates.
 
 ## Further reading
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /product_analytics/journeys/funnel_analysis
 [2]: /product_analytics/journeys/sankey
-[3]: https://app.datadoghq.com/product-analytics/user-journey/conversion?
\ No newline at end of file
+[3]: https://app.datadoghq.com/product-analytics/user-journey/conversion?
```

Comment:
This link redirects to `https://app.datadoghq.com/product-analytics/user-journey/funnel?`. Is this correct?